rust/operator-binary/src/authorization/opa.rs

```diff
@@ -39,6 +43,27 @@ impl TrinoOpaConfig {
                 opa_authorizer_name: "tech.stackable.trino.opa.OpaAuthorizer".to_string(),
                 non_batched_connection_string,
                 batched_connection_string: None,
+                allow_permission_management_operations: false,
+            })
+        } else if PRODUCT_VERSIONS_WITH_INTERMEDIATE_AUTHORIZER
+            .contains(&resolved_product_image.product_version.as_str())
+        {
+            let non_batched_connection_string = opa_config
+                .full_document_url_from_config_map(client, trino, Some("allow"), OpaApiVersion::V1)
+                .await?;
+            let batched_connection_string = opa_config
+                .full_document_url_from_config_map(
+                    client,
+                    trino,
+                    Some("extended"),
```

Comment:
I would like to make a switch from `extended` to `batchAllow` (more descriptive name), it would be great if we could to that - as the rego rules are now breaking anyway

Reply:
done